core/controller/src/main/scala/whisk/core/entitlement/ActivationThrottler.scala

```diff
@@ -48,7 +48,7 @@ class ActivationThrottler(loadBalancer: LoadBalancer, defaultConcurrencyLimit: I
   def check(user: Identity)(implicit tid: TransactionId): Future[RateLimit] = {
     loadBalancer.activeActivationsFor(user.uuid).map { concurrentActivations =>
       val concurrencyLimit = user.limits.concurrentInvocations.getOrElse(defaultConcurrencyLimit)
-      logging.info(
+      logging.debug(
```

Comment:
these proved helpful in the past to know if a uuid was over its quota. can we revert?

Reply:
great that will be helpful, is that in this pr or subsequent pr?

Reply:
ok i see it later on; see related comment there.